src/test/java/pro/sky/employees/service/EmployeesServiceTest.java

```diff
@@ -21,37 +25,78 @@ public class EmployeesServiceTest {
 
     @BeforeEach
     public void beforeEach() {
-        employees.forEach(employee -> employeeService.add(
-                employee.getFirstName(),
-                employee.getLastName(),
-                employee.getSalary(),
-                employee.getDepartmentId()));
+        employees.forEach(employee -> employeeService.add(employee.getFirstName(), employee.getLastName(), employee.getSalary(), employee.getDepartmentId()));
     }
 
     @AfterEach
     public void afterEach() {
-        employeeService.findAll()
-                .forEach(Employee -> employeeService.remove(Employee.getFirstName(), Employee.getLastName()));
+        employeeService.findAll().forEach(Employee -> employeeService.remove(Employee.getFirstName(), Employee.getLastName()));
```

Comment:
```suggestion
        employeeService.findAll().forEach(employee -> employeeService.remove(employee.getFirstName(), employee.getLastName()));
```
названия переменных с маленькой буквы пишем